lib/utils.js

```diff
@@ -0,0 +1,15 @@
+const gitLogParser = require('git-log-parser')
+const toArray = require('stream-to-array')
```

Comment:
Could we use https://www.npmjs.com/package/get-stream instead? It seems better maintained.

Reply:
Sure I can do that!
The reason I went with `stream-to-array` was because of the example/recommend way of doing it from `git-log-parser`